src/components/app/details/appDetails/AppDetails.tsx

```diff
@@ -425,178 +423,178 @@ export const Details: React.FC<{
-                externalLinks={externalLinks}
-                monitoringTools={monitoringTools}
-                appDetails={appDetails}
-            />
-            <ResponsiveDrawer
-                className="events-logs"
-                isDetailedView={!!params.tab}
-                onHeightChange={(height) => (document.getElementById('dummy-div').style.height = `${height}px`)}
-                anchor={params.kind ? <EventsLogsTabSelector /> : null}
-            >
-                <Route path={`${path.replace('/:kind?', '/:kind').replace('/:tab?', '/:tab')}`}>
-                    <NodeSelectors
-                        isAppDeployment={isAppDeployment}
-                        selectedContainer={selectedContainer}
-                        logsPaused={logsPaused}
-                        logsCleared={logsCleared}
-                        socketConnection={socketConnection}
-                        containerName={selectedContainer}
-                        nodeName={selectedNode}
-                        selectedNodes={selectedNodes}
-                        nodeItems={nodeItems}
-                        nodes={nodes}
-                        shell={shell}
-                        isReconnection={isReconnection}
-                        setIsReconnection={setIsReconnection}
-                        setLogsCleared={setLogsCleared}
-                        selectShell={selectShell}
-                        setSocketConnection={setSocketConnection}
-                        setTerminalCleared={setTerminalCleared}
-                        handleLogsPause={handleLogsPause}
-                        selectNode={selectNode}
-                        setSelectNode={setSelectNode}
-                        selectContainer={selectContainer}
-                    />
-                    <EventsLogs nodeName={selectedNode}
-                        isAppDeployment={isAppDeployment}
-                        selectedLogsNode={selectedNodes}
-                        nodeItems={nodeItems}
-                        logsCleared={logsCleared}
-                        nodes={nodes}
-                        appDetails={appDetails}
-                        containerName={selectedContainer}
-                        logsPaused={logsPaused}
-                        socketConnection={socketConnection}
-                        terminalCleared={terminalCleared}
-                        shell={shell}
-                        isReconnection={isReconnection}
-                        setIsReconnection={setIsReconnection}
-                        selectShell={selectShell}
-                        setTerminalCleared={setTerminalCleared}
-                        setSocketConnection={setSocketConnection}
-                        handleLogPause={handleLogsPause}
-                    />
-                </Route>
-            </ResponsiveDrawer>
-            <div id="dummy-div" style={{ width: '100%', height: '32px' }}></div>
-        </>
-    );
-};
+// const NodeDetails: React.FC<{
```

Comment:
Please remove commented code if not required

Reply:
Removed